mobile-app/app/components/WalletAccordion.tsx

```diff
@@ -26,7 +27,7 @@ interface ActiveSessions {
 export function WalletAccordion (props: AccordionProps): JSX.Element {
   const { isLight } = useThemeContext()
   const initialContent: ActiveSessions = {
-    activeSessions: []
+    activeSessions: props.activeSessions ?? []
```

Comment:
While we're at it, can we rename it?
```suggestion
    activeSections: props.activeSections ?? []
```